mir-ci/readme.md

```diff
@@ -1,6 +1,6 @@
 # Ubuntu Frame Integration Tests
 
-**NB:** Your working directory needs to be `mir-ci` for this to work.
+**NB:** Your working directory needs to be `mir_ci` for this to work.
```

Comment:
Unrelated to this line, but a good place to put it:

Should we add a command to `lint` the whole project? Maybe a script or just directions in the README?

Reply:
I'll sleep on it, maybe work out a pre-commit hook.